deployer/src/main/java/brooklyn/entity/cloudfoundry/webapp/CloudFoundryWebAppImpl.java

```diff
@@ -0,0 +1,253 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package brooklyn.entity.cloudfoundry.webapp;
+
+
+import brooklyn.entity.Entity;
+import brooklyn.entity.annotation.Effector;
+import brooklyn.entity.annotation.EffectorParam;
+import brooklyn.entity.cloudfoundry.CloudFoundryEntityImpl;
+import brooklyn.event.feed.function.FunctionFeed;
+import brooklyn.event.feed.function.FunctionPollConfig;
+import brooklyn.util.collections.MutableMap;
+import brooklyn.util.time.Duration;
+import com.google.common.base.Functions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.LinkedList;
+import java.util.Map;
+import java.util.concurrent.Callable;
+
+public abstract class CloudFoundryWebAppImpl extends CloudFoundryEntityImpl
```

Comment:
I am not sure. This first commit contains a first approach our work about CF integration in SeaClouds. As I mentioned previously @mbarrientos and I are working in the CF integration in Brooklyn. But I think it will be a large process because we are changing several aspect that should be discuss.
I'd prefer adding this commit and in the future changing the necessary entities.